### ISSUE TITLE
Add a History page?

### ISSUE THREAD

**Issue by Beliavsky** [fortran-lang/fortran-lang.org#218]
Fortran-lang.org promotes Fortran as a modern language, which it is. But it's also an old language with a rich history that some site visitors may be interested in. At the top, along with the sections 

Learn
Compilers
Community
Packages
News

should there be a link to a History page with relevant information?

**Comment by arjenmarkus** [comment]
I definitely like the idea. It might the place to show the dichotomy
between "ancient" FORTRAN and modern Fortran.

Op do 4 mrt. 2021 om 00:01 schreef Beliavsky <notifications@github.com>:

**Comment by certik** [comment]
Yes I like the idea. We should have a History section.

**Comment by awvwgk** [comment]
I like the idea as well. Having a good organized knowledge base over the history of Fortran would be a useful addition. Creating such a new section with meaningful content can be more difficult and requires expertise and input from long practitioners and pioneers of Fortran.

Input and patches are very welcome on this. I'm happy to help where ever I can with this, but I'm out for doing the first steps on this (I started programming Fortran in 2017, not the best qualifications for writing a history section).

**Comment by vmagnin** [comment]
My feeling is that a Fortran history page in Fortran-lang.org should be designed with clear and assumed goals. There is already a lot of pages with a Fortran history (for example https://en.wikipedia.org/wiki/Fortran).
Our history should not be another old dusty museum but concentrate on events and information that could be useful for Fortran beginners to understand the present state of Fortran:
- why Fortran was created, why it was successful (easy to learn, a compiler designed to achieve good optimizations since the first version, oriented for array computing, etc.),
- why it was standardized, how standardization is functioning since Fortran 90 (major/minor versions, many years between the standard and the implementation in compilers...) It's very different from younger languages where you would have Fortran 3.0, Fortran 3.1, Fortran 3.1.5...
- Why most modern Fortran files have a .f90 extension,
- insist on the perenniality of Fortran codes,
- Etc.

And we have already many posts in the Discourse that could bring materials to build such a story.

**Comment by milancurcic** [comment]
I agree, this would be a good addition to the website. I think a mini-book would be a good format.

**Comment by vmagnin** [comment]
This Curlie directory (the historical DMOZ reborn) list a few pages with stuff related to Fortran history:
https://www.curlie.org/en/Computers/Programming/Languages/Fortran/History

And especially this page:
http://www.softwarepreservation.org/projects/FORTRAN/

**Comment by Beliavsky** [comment]
Related discussion at Fortran Discourse: [History of Fortran (Fortran II source code)](https://fortran-lang.discourse.group/t/history-of-fortran-fortran-ii-source-code/117)

**Comment by vmagnin** [comment]
Another discussion: 
https://fortran-lang.discourse.group/t/john-backus-1924-2007-fortrans-father/873
